src/goGraph/node-templates/symbol-node-template.ts

```diff
@@ -1,19 +1,121 @@
-import go from 'gojs';
+import go, { GraphObject } from 'gojs';
 import { portDirectionToSpot } from '../item-templates/position-port-item-template';
 
+// this is shown by the mouseHover event handler
+var nodeHoverMenu = new go.Adornment('Spot', {
```

Comment:
This file is getting big. Can we separate nodeHoverMenu into separate file, with an exporting function returning it?

Reply:
I will tidy up this file in a later PR

Reply:
👍